src/main/webapp/sass/_core.scss

```diff
@@ -3888,3 +3888,23 @@ div.border {
   margin: 0px;
   padding: 15px;
 }
+
+// swc.css list styles override MUI list styles
+// Re-specify MUI list style here
+.accessRequirementRelatedProjectsListContainer > div {
```

Comment:
Override the swc.css list styles, so that styles match MUI

Reply:
This works for now knowing how challenging it is to get MUI to work with bootstrap, normalize, and anything custom we have in SWC